src/Symfony/Component/Dotenv/README.md

```diff
@@ -11,6 +11,13 @@ Getting Started
 composer require symfony/dotenv
 ```
 
+Usage
+---------------
+
+```env
```

Comment:
If we add this code block, it need to be introduced.

> For an `.env` file with this format:

Reply:
@GromNaN  Added